README.md

```diff
@@ -1,2 +1,55 @@
 # axiom-beta-qemu
 QEMU emulation of the AXIOM Beta hardware / software
+
+This scripts will help you to run AXIOM Beta OS on QEMU.
+Follow the following steps to build and run the QEMU.
+
+# Step 1: Build and Installing QEMU 
+First you need to build and install the QEMU. For that, go open the terminal on axiom-beta-qemu repo folder and run the following commands.
+
+`chmod +x build_qemu.sh`
+
+`./build_qemu.sh`
+
+Note: This will download QEMU form Xilinx QEMU repo. Therefore you need to have internet conneciton while running this script. Also provide system password to install QEMU.
+
+# Step 2-[A]: Build Xilinx Kernel 4.6.0
+Note: You need to follow either Step 2-[A] or 2-[B].
+If you are hope to run Kernel 4.6.0, then run this step. If you are hoping to run Kernel4.9.0, then follow step 2-[B]. However, as we tested, Kernel4.6.0 work properly on QEMU.
```

Comment:
> If you are hope to
> Do you mean If you hope you are able to...?
I mean, you can use either kernel 4.6.0 or 4.9.0. Step 2-[A] is to kernel 4.6.0 and 2-[B] is to kernel 4.9.0

will update the other mistakes shortly.